templates/errors/submission-complete.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
This page has an exit button which doesn't do anything.

Reply:
I can't see a button when testing it? 
![image](https://user-images.githubusercontent.com/47635349/92693968-3f13b700-f33e-11ea-88d1-d6e68023a472.png)

Reply:
It will show up when your session hasn't expired. So try what you did but on a diff schema that doesn't timeout that quick.

Reply:
I've added a fix for this